src/Hangman.java

```diff
@@ -0,0 +1,82 @@
+import java.util.List;
+
+public class Hangman {
+
+    private List<Character> characters;
```

Comment:
Like I've said before. Initialize those fields here. Do not pass them to constructor as params

Reply:
I don't think we need any getters and setters in this class since whole logic should be here. No outside class should have access to any fields of this class